test/feature/support/helpers.coffee

```diff
@@ -19,12 +19,21 @@ class Helpers
       app.use exprestive(#{optionsStr})
       app.listen #{@port}
 
+      # Route to evaluate functions from tests
+      app.get '/eval/:strToEval', (req, res) ->
```

Comment:
I mean that the routes file can contain something like

```
GET '/help/:topic', to: 'help#index'
```

And then if I go to /help/foo it goes to HelpController.index and the `params` hash contains `topic: 'foo'`.